### PR TITLE
Implement right-click functionality for crops

### PR DESCRIPTION
**Pull request by Ethryan**
For all the crops.

### CODE REVIEW

**Review by Alastors — APPROVED** [review]
Sure, looks fine, though I swear it already had this.

File: src/main/java/tb/common/block/BlockTBPlant.java [hunk 1]
```diff
@@ -140,6 +143,43 @@ public void func_149853_b(World w, Random r, int x, int y, int z) {
...
+                            fortune = heldItem.getEnchantmentTagList()
+                                .getCompoundTagAt(i)
+                                .getInteger("lvl");
+
+                            // Damage fortune item on crop break?
+                            // heldItem.setItemDamage(heldItem.getItemDamage() + 1);
+                            break;
+                        }
+                    }
+                }
```
Comment (OneEyeMaker) [inline src/main/java/tb/common/block/BlockTBPlant.java:171]: This can easily be done with `EnchantmentHelper`.

**Review by OneEyeMaker — CHANGES_REQUESTED** [review]
Please, use method `EnchantmentHelper.getFortuneModifier`.

File: src/main/java/tb/common/block/BlockTBPlant.java (hunk 1 above)
Comment (Ethryan) [reply src/main/java/tb/common/block/BlockTBPlant.java:171]: Working on it.

**Review by OneEyeMaker — APPROVED** [review]

**Pull request merged**